src/App.js

```diff
@@ -1,24 +1,27 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import {
+  BrowserRouter as Router, Route, Link,
+} from 'react-router-dom';
+import BookList from './components/Booklist';
+import Categories from './components/Categories';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div>
+        <nav>
+          <ul>
+            <li><Link to="/">Home</Link></li>
+            <li><Link to="/categories">Categories</Link></li>
+          </ul>
+        </nav>
+
+        <div>
+          <Route exact path="/" component={BookList} />
+          <Route path="/categories" component={Categories} />
+        </div>
```

Comment:
`Routing` is cool when done the right way 🚗 your routing is throwing an error for now ❌ 😢  this is what it says 👇🏽 
```
Uncaught Error: 
A <Route> is only ever to be used as the child of <Routes> element, never rendered directly. 
Please wrap your <Route> in a <Routes>.
```
The issue seems to be linked to the fact that you wrapped your `Route` with `div` instead of `Routes`, this is what the console gives me 👇🏽 

![Screenshot 2023-04-08 at 19 15 50](https://user-images.githubusercontent.com/45739963/230746333-917990b3-e1cb-47f8-978d-e07acdb06bad.png)